src/collector/rrdp.rs

```diff
@@ -1997,6 +1989,51 @@ impl RepositoryObject {
         reader.into_hash().compose(target)?;
         Ok(())
     }
+
+    /// Dumps an object to the given target path.
+    ///
+    /// The dumped object will only contain the real object data, not the
+    /// added hash.
+    pub fn dump(
+        source_path: &Path,
+        target_path: &Path,
+    ) -> Result<(), Failed> {
+        let mut source = match File::open(source_path) {
+            Ok(source) => source,
+            Err(err) => {
+                error!(
+                    "Failed to open source RRDP file {}: {}",
```

Comment:
Consistency with other error messages:
```suggestion
                    "Failed to open RRDP source file {}: {}",
```

Reply:
I kinda like my own version better: This is really is about the source of the RRDP file to be dumped, not about the RRDP source of a file.